srcpkgs/python3-Twisted/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'python3-Twisted'
+pkgname=python3-Twisted
+version=22.1.0
+revision=1
+wrksrc="Twisted-${version}"
+build_style=python3-module
+make_check_target=src/twisted
+hostmakedepends="python3-setuptools python3-incremental"
+makedepends="python3-devel"
+depends="python3-zope.interface python3-constantly python3-incremental
+ python3-automat python3-hyperlink python3-attrs python3-typing_extensions
+ python3-openssl python3-service_identity python3-pyasn1"
+checkdepends="python3-bcrypt python3-PyHamcrest git $depends"
+short_desc="Asynchronous networking framework written in Python (Python3)"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://twistedmatrix.com/"
+changelog="https://raw.githubusercontent.com/twisted/twisted/trunk/NEWS.rst"
+distfiles="${PYPI_SITE}/T/Twisted/Twisted-${version}.tar.gz"
+checksum=b7971ec9805b0f80e1dcb1a3721d7bfad636d5f909de687430ce373979d67b61
+make_check=ci-skip # some tests fail when running as root
+
+alternatives="
+ twisted:cftp:/usr/bin/cftp3
+ twisted:ckeygen:/usr/bin/ckeygen3
+ twisted:conch:/usr/bin/conch3
+ twisted:mailmail:/usr/bin/mailmail3
+ twisted:pyhtmlizer:/usr/bin/pyhtmlizer3
+ twisted:tkconch:/usr/bin/tkconch3
+ twisted:trial:/usr/bin/trial3
+ twisted:twist:/usr/bin/twist3
+ twisted:twistd:/usr/bin/twistd3"
+
+post_patch() {
+	# test requires unpackaged cython-test-exception-raiser
+	rm src/twisted/test/test_failure.py
+}
+
+do_check() {
+	python3 setup.py install --prefix=build --root="$(pwd)"
+	PATH="$(cd build/bin && pwd)":$PATH \
+		PYTHONPATH="$(cd build/lib/python*/site-packages && pwd)" \
```

Comment:
```suggestion
		PYTHONPATH="$PWD/build/lib/python$py3_ver/site-packages" \
```
maybe $PWD is not required

Reply:
I tried without $PWD and had failing tests